code/game/objects/items/toys.dm

```diff
@@ -1536,3 +1538,130 @@
+	var/usable = FALSE
+	/// Associated list of intents to their sounds
+	var/static/list/sound_by_intent = list(
+		HELP = 'sound/items/intents/Help.ogg',
+		DISARM = 'sound/items/intents/Disarm.ogg',
+		GRAB = 'sound/items/intents/Grab.ogg',
+		HARM = 'sound/items/intents/Harm.ogg',
+		)
+
+/obj/item/toy/intento/emag_act(mob/user)
+	if(obj_flags & EMAGGED)
+		return
+	obj_flags |= EMAGGED
+	to_chat(user, "<span class='notice'>You short-circuit [src], activating the negative feedback loop.</span>")
+
+/obj/item/toy/intento/attack_self(mob/user, modifiers) //added params to attack_self, the alternative is registering a signal on clickon but i was advised not to
+	..()
+	if(!on)
+		start()
+		return
+	if(!modifiers || !usable)
+		return
+	usable = FALSE
+	var/input
+	var/icon_x = text2num(modifiers["icon-x"])
+	var/icon_y = text2num(modifiers["icon-y"])
+	if(icon_x > ICON_SPLIT && icon_y > ICON_SPLIT)
+		input = DISARM
+	if(icon_x < ICON_SPLIT && icon_y > ICON_SPLIT)
+		input = HELP
+	if(icon_x > ICON_SPLIT && icon_y < ICON_SPLIT)
+		input = GRAB
+	if(icon_x < ICON_SPLIT && icon_y < ICON_SPLIT)
+		input = HARM
+	render(input)
+	player_sequence += input
+	for(var/i in 1 to player_sequence.len)
+		if(player_sequence[i] != current_sequence[i])
+			return end(user)
+	if(player_sequence.len == current_sequence.len)
+		score++
+		addtimer(CALLBACK(src, .proc/start_round, 1 SECONDS))
+		return
+	usable = TRUE
+
+/obj/item/toy/intento/proc/start()
+	say("Game starting!")
+	playsound(src, 'sound/machines/synth_yes.ogg', 50, FALSE)
+	on = TRUE
+	addtimer(CALLBACK(src, .proc/start_round, 1 SECONDS))
+
+/obj/item/toy/intento/proc/end(mob/user)
+	var/award_score = score
+	var/award_status = user.client.get_award_status(/datum/award/score/intento_score)
+	if(award_score - award_status > 0)
+		award_score -= award_status
+	user.client.give_award(/datum/award/score/intento_score, user, award_score, TRUE)
+	say("GAME OVER. Your score was [score]!")
+	playsound(src, 'sound/machines/synth_no.ogg', 50, FALSE)
+	if(obj_flags & EMAGGED)
+		retaliate(current_sequence.Copy(), user)
+	score = 0
+	player_sequence.Cut()
+	current_sequence.Cut()
+	on = FALSE
+
+/obj/item/toy/intento/proc/render(input)
+	icon_state = input
+	playsound(src, sound_by_intent[input], 50, FALSE)
+	addtimer(VARSET_CALLBACK(src, icon_state, initial(icon_state)), 0.5 SECONDS)
+
+/obj/item/toy/intento/proc/start_round()
+	usable = FALSE
+	player_sequence.Cut()
+	current_sequence += pick(list(HELP, DISARM, GRAB, HARM))
+	for(var/input in current_sequence)
+		sleep(0.6 SECONDS)
+		render(input)
+	usable = TRUE
+
+/obj/item/toy/intento/proc/retaliate(list/combo_inputs, mob/living/victim)
+	ADD_TRAIT(src, TRAIT_NODROP, "intento")
+	to_chat(victim, "<span class='userdanger'>Bad mistake.</span>")
+	for(var/input in combo_inputs)
+		sleep(0.6 SECONDS)
+		render(input)
+		switch(input)
+			if(HELP)
+				to_chat(victim, "<span class='danger'>[src] hugs you to make you feel better!</span>")
+				victim.Dizzy(20)
```

Comment:
I kind of like the idea of the help parts of the combo just giving you a positive hug moodlet and not actually debuffing you; would you mind replacing the Dizzy() part of the combo here with that?

also, IIRC, Dizzy uses seconds, not deciseconds, so this lasts 10x as long as most of these other effects

Reply:
sure